03week/airplane/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<link rel="stylesheet" href="css/style.css" />
+
+<body>
+  <header>
+    <span>
+          <img src="img/logo.png" alt="" height=70px>
+        </span>
+    <nav>
+      <img src="img/home-icon.png" alt="">
```

Comment:
These navigation elements are a little too close together. You might consider adding padding to them